plugins/orchestrator-backend/src/service/SonataFlowService.ts

```diff
@@ -237,4 +241,8 @@ export class SonataFlowService {
     }
     return false;
   }
+
+  public getFetchErrorMessage(urlToFetch: string, responseStr: string): string {
+    return `Response was NOT okay when fetch(${urlToFetch}). Received response ${responseStr}`;
```

Comment:
Response was NOT okay => Response status was NOT successful

Reply:
Addressed